public/static-checkout/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Payment | edX</title>
+  <link rel="shortcut icon" type="image/x-icon" href="assets/favicon.ico"/>
+  <link href="css/style.css" rel="stylesheet" type="text/css">
```

Comment:
this should probably be minified and directly included here, but we could do that when we're ready